trade-finance-manager-ui/templates/case/amendments/_macros/status-tag.njk

```diff
@@ -15,6 +15,15 @@
     {% set statusClass = 'govuk-tag--green' %}
   {% endif %}
 
+  {% if decision === 'Proceed' %}
```

Comment:
for amendments work, can we replace this file with something that works like this
```
{{govukTag({
  text: amendment.bankDecision.decision,
  classes: amendment.tags[amendment.bankDecision.decision],
  attributes: attributes: { "data-cy": "amendment--details-"+amendment.version+"-banks-decision" }
})}}
```